### ISSUE TITLE
Could you explain "select sentences according to the logits changed"?

### ISSUE THREAD

**Issue by FlyCarrot** [zhiyuanhubj/LongRecipe#1]
Hi, it's a good code repo, but I don't find the code for calculating logits changed, Could you point to the target code line? 
Thanks a lot!

**Comment by zmwang03** [comment]
Same issue

**Comment by zhiyuanhubj** [comment]
Hi @FlyCarrot @IanBestFellow, thanks for your interest in our work. We have modified the relevant code in vllm for more efficient computation and have attached our code, along with the readme for implementation, [here](https://github.com/zhiyuanhubj/LongRecipe/tree/main/utils/logits_compute).

We hope this addresses your issues. Feel free to post any further questions."

Let me know if this works for you!
